core/geometry/src/geometry3d/PolylineOps.ts

```diff
@@ -189,5 +189,24 @@ export class PolylineOps {
       (data as Point3d[]).push(q0.clone());
     }
   }
-
+  /**
+   * Remove closure points a polyline or array of polylines
+   * @param data points.
+   */
+   public static removeClosurePoint(data: Point3d[] | Point3d[][]) {
+    if (data.length === 0)
+      return;
+    const q0 = data[0];
+    if (Array.isArray(q0)) {
+      for (const child of data) {
+        if (Array.isArray(child))
+          this.addClosurePoint(child);
```

Comment:
Should be `this.removeClosurePoint(child)`

Reply:
YES !!!

Reply:
(fix pushed)